### DISCUSSION TITLE
Importazione voti scrutinio

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#634] [Idee e proposte]
Buonasera ,allego questo link di un video youtube di durata 44 secondi inerente a una cosa che renderebbe questo registro veramente di alto livello ossia l'importazione dei voti per gli scrutini : https://www.youtube.com/watch?v=XYUhVwdabY0

**Comment by Davide031** [comment]
esatto sarebbe molto più facile fare gli scrutini

**Comment by sempliciniluigi89** [comment]
Ci metteremmo  5/10 minuti per classe ,dal momento che il voto
scatturirebbe dalla media dei voti accumulati nei singoli periodi

Il giorno mar 23 gen 2024 alle ore 20:42 Davide031 ***@***.***>
ha scritto:

**Comment by trinko** [comment]
Se la tua idea di registro ideale è Argo, allora rassegnati: giua@school non diventarà mai simile a quella roba lì... Ma ti rendi conto che per inserire una solo funzionalità devi fare una decina di click ed effettuare altrettante scelte nei parametri? E' una follia, tant'è vero che tutte le scuole sono costrette a fare dei tutorial per spiegare quella che dovrebbe essere una cosa semplice...

Poi, si può anche migliorare la gestione delle proposte di voto, ma lasciamo perdere Argo, per carità.

**Reply by Davide031** [reply]
Vero, acnhe se argo è un ottimo registro a questione di grafica ma per me è diecimila volte meglio giua@school perchè è molto più semplice e dico davvero
